helx/charts/nginx/templates/ingress.yaml

```diff
@@ -25,4 +25,32 @@ spec:
             name: {{ include "nginx.fullname" . }}
             port:
               number: {{ .Values.service.httpPort }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "nginx.fullname" . }}-admin
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: "{{ .Values.ingress.admin_ip_whitelist }}"
```

Comment:
Could we wrap this line in an if statement? The reason is that if this annotation is present at all (even if it's set to `""`), that will override any global whitelist to allow all traffic: https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#whitelist-source-range

```
{{- if .Values.ingress.admin_ip_whitelist }}
      nginx.ingress.kubernetes.io/whitelist-source-range: "{{ .Values.ingress.admin_ip_whitelist }}"
{{- end}}
```

Reply:
Wrapping the entire ingress definition with that if statement as in [this comment](https://github.com/helxplatform/devops/pull/148/files#r720343706) would achieve that, right?

Reply:
Ah yes, will resolve this comment then